test/agoneum/web/controllers/registration_controller_test.exs

```diff
@@ -1,30 +1,47 @@
 defmodule Agoneum.Web.RegistrationControllerTest do
   use Agoneum.Web.ConnCase
 
-  alias Agoneum.Account
+  describe "UI registration" do
```

Comment:
I'm not very good with the difference between those. How would that work with both UI and API tests?